test/parallel/test-fs-mkdir.js

```diff
@@ -316,7 +316,7 @@ if (common.isMainThread && (common.isLinux || common.isOSX)) {
   const p = fs.mkdirSync(pathname, common.mustNotMutateObjectDeep({ recursive: true }));
   assert.strictEqual(fs.existsSync(pathname), true);
   assert.strictEqual(fs.statSync(pathname).isDirectory(), true);
-  assert.strictEqual(p, firstPathCreated);
+  assert.strictEqual(p, path.toNamespacedPath(firstPathCreated));
```

Comment:
Can you explain why we are changing the comparison to include the `path.toNamespacedPath`? Did `tmpdir.resolve` API change somehow? If so, isn't it a breaking change?

Reply:
Technically it isn't since both paths are the same and from Windows perspective having a path with namespaced value does not really have any difference with without the prefix. 

PS: Namespaced paths are used to avoid "path is too long" errors on Windows. If you look into this file, you'll see that we already check the value surrounded by toNamespacedPath in several places already. I feel like this is more complete.